src/schema/types/product.graphql

```diff
@@ -13,6 +14,7 @@ input ProductInput {
   description: String
   price: Float!
   marketPrice: Float!
+  productPhotos: [String!]
```

Comment:
Shouldn't this be productPhotosUrls?

Reply:
But these strings are the base64 images. Basically it's just strings. Should I create a scalar type?

Reply:
Oh! You mean the property name!